src/commands/pull.js

```diff
@@ -0,0 +1,69 @@
+const chalk = require('chalk');
+const mkdirp = require('mkdirp');
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const { ApifyCommand } = require('../lib/apify_command');
+const { getLoggedClientOrThrow } = require('../lib/utils');
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args } = this.parse(PullCommand);
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const { actorId } = args;
+        const actor = await apifyClient.actor(actorId).get();
+        if (!actor) throw new Error(`Cannot find actor with ID '${actorId}' in your account.`);
+        const { name, versions } = actor;
+
+        // TODO: Check is sorted chronologically
+        const lastVersion = versions[versions.length - 1];
+
+        const dirpath = path.join(cwd, name);
+        mkdirp.sync(dirpath);
```

Comment:
It would be nice if it checked that the directory is not empty, and either errored out or prompted you to overwrite it, or merge the files. Right now it will just merge the files without asking, which is not what you want by default (e.g. files deleted in the Console will not be deleted after `apify pull`).